data/discord

```diff
@@ -3,8 +3,16 @@ discord.com
 discord.gg
 discord.gift
 discord.gifts
-discord.media
```

Comment:
Why is `discord.media` removed? Is still a Discord domain and is used for voice servers 😋

Reply:
Well... i forgot to add it again xD